.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  release:
+    types: [published]
```

Comment:
This will run the workflow any time a new release is published. There are a few event types here listed [here](https://docs.github.com/en/actions/reference/events-that-trigger-workflows), but I think published makes the most sense.